test/test_event.py

```diff
@@ -3,7 +3,11 @@
 import time
 
 def test_event_default_time():
-    time.sleep(1.5)
+    """ensure created_at default value reflects the time
+    at Event object instantiation 
```

Comment:
Formatting nits:
* I tend to be a bit loose-ish with PEP-8 when it comes to comments. I generally aim for a line break at char 90 instead of the (I think) recommended 80. To me this comment line is close enough that I'd probably keep it as a one-liner.
* However, if a triple-quoted comment is multiline, I usually format as:
    ```python
    """
        My multline comment. My multline comment. My multline 
        comment. My multline comment.
    """
    ```

    with the triple quotes on their own line. I don't know if PEP-8 has any opinions on that, though.